projects/project-1/assignment/assignment_1_short.md

```diff
@@ -0,0 +1,3235 @@
+
+B/Ai
+If B, inverse of A, is symmetric it means that yAx and xAy both hold. This contradicts Ai´s asymmetry, where xAiy forbids yAix. making it impossible for B to be symmetric and Ai asymmetric.
+
+B/B:
+same as A/A
+
+B/Bi:
+Bi as the inverse of B can conflict if B is symmetric or asymmetric. The inverse relationship can cause contradictions with symmetry or asymmetry (X).
+
+B/Ai:
+If xAy implies yAx (symmetry) or ¬(yAx) (asymmetry), then B as a sub-property of A and Ai​ as its inverse can conflict because xBy and xAi​y may not align due to the inverse relationship (X).
+
+Bi/Bi:
+same as A/A
+
+
+- Trans/Asymm
+
+- <img width="639" alt="image" src="https://github.com/user-attachments/assets/f713b95b-ed69-413e-bc04-1580acf8bb98">
+
+A/A:
+See exercise 1
+
+- For A/Ai, suppose that A is transitive and Ai is asymmetric. Being Ai the inverse of A, it is the case that the relation A holds between the opposite instances in the relation Ai. But this creates a complex role chain (see exercise 1).
+  
+- For Ai/A, suppose that Ai is transitive and A is asymmetric. Being A the inverse of Ai, it is the case that the relation Ai holds between the opposite instances in the relation A. But this creates a complex role chain (see exercise 1).
+  
+- For Ai/Ai, see Exercise 1
+  
+- For B/A, suppose that B is transitive and A is asymmetric. Being B a subproperty of A, xBy implies xAy. But this creates a complex role chain (see exercise 1).
+  
+-  For B/B, see Exercise 1
+
+- For B/Bi, suppose that B is transitive and Bi is asymmetric. Being Bi the inverse of B, it is the case that the relation B holds between the opposite instances in the relation Bi. But this creates a complex role chain (see exercise 1).
+
+- For Bi/A, suppose that Bi is transitive and A is asymmetric. Being Bi the inverse of B, it is the case that the relation B holds between the opposite instances in the relation Bi. Being B a subproperty of A, xBy implies xAy. But this creates a complex role chain (see exercise 1).
+
+- For Bi, Ai, suppose that Bi is transitive and Ai is asymmetric. Being Bi the inverse of B, it is the case that the relation B holds between the opposite instances in the relation Bi. Being B a subproperty of A, xBy implies xAy. Being A the inverse of Ai, it is the case that the relation Ai holds between the opposite instances in the relation A. But this creates a complex role chain (see exercise 1).
+
+-  For Bi/Bi, see Exercise 1
+
+
+- Reflexivity/Asymmetry
+
+<img width="643" alt="image" src="https://github.com/user-attachments/assets/cf5ae51e-1d23-4001-a15a-aa67e25e137c">
+
+
+- For A/A, Ai/Ai, B/B, Bi/Bi: see Exercise 1
+
+- A/Ai: suppose that A is reflexive and Ai is asymmetric. By reflexivity, xAx. Since Ai is inverse A, xAix also holds.
+However, because of asymmetry, xAix should not hold.   
+
+- A/B: OK
+
+- A/Bi: Ok
+
+- Ai/A: suppose that Ai is reflexive and A is asymmetric. By reflexivity, xAix. Since A is inverse Ai, xAix also holds.
```

Comment:
You use both "asymmetricity" and "asymmetry," but asymmetry is the standard term. Stick to "asymmetry" for consistency

Reply:
There’s a slight ambiguity with the phrase "because of asymmetricity, xAx should not hold"
I have tried to reformulate it in this way: suppose Ai is reflexive and A is asymmetric. Reflexivity implies that xAix. Since A is the inverse of Ai, this also means that xAx must hold. However, asymmetry implies that if xAx, then xAix cannot hold, leading to a contradiction.